apps/docs/content/components/Heading/index.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Heading
+description: Use the Heading component to render title or subtitle text
+---
+
+import CustomVideoPlayer from '../../../src/components/custom-video-player'
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+![An image displaying each header from h1 to h6.](https://github.com/primer/brand/assets/912236/daf645d9-2ce5-4aba-9bc0-7c95b401c6c8)
+
+## Usage
+
+Headings are used to structure the content on the page, with each heading representing a different level of importance. It is recommended to use headings in a logical manner to create a clear hierarchy and improve accessibility.
+
+- Heading should be used to communicate page organization and hierarchy.
+- Use heading as the title that introduces a page, a section or an article.
+- The heading component will render an html h2 tag without any default styling.
+
+## Options
+
+### Scale and size
+
+The Heading scale ranges from 1-6 and maps to the HTML heading elements h1-h6 respectively. The size is determined by the type of HTML element specified, although you can override this when the scenario calls for it. E.g. use a `h4` heading size on a `<h3>` to highlight an article that might be relevant to the user.
```

Comment:
```suggestion
The Heading scale ranges from 1 to 6 and maps to the HTML heading elements h1 to h6 respectively. The size is determined by the type of HTML element specified, although you can override this when the scenario calls for it. E.g. use a `h4` heading size on a `<h3>` to highlight an article that might be relevant to the user.
```
Nitpick

Reply:
> E.g. use a `h4` heading size on a `<h3>` to highlight an article that might be relevant to the user.

I'm not sure I understand the example correctly. Isn't it the other way around? If we want to highlight something, shouldn't we use a heading size that is larger than its semantic element's size?

Reply:
true! It should be `h2` heading size instead of `h4`